### ISSUE TITLE
SPIKE: Subject headings are displayed in record-view as segments (facets), rather than as full strings

### ISSUE THREAD

**Issue by motropuk** [yalelibrary/YUL-DC#1805]
**Story**

Subject headings in DCS are currently being broken out on the digital object page as segments, rather than as full strings. We need to update this to display strings instead. 

Proposed solution is to match how this is handled in QuickSearch. Details here https://git.yale.edu/Library-IT/Metadata_Field_Mappings/issues/32. Solution is here https://git.yale.edu/Library-IT/Metadata_Field_Mappings/issues/32#issuecomment-12440  

**Acceptance**
- [x] Spike:  Figure out approach
- [x] Create new ticket that asks for this work to be completed (re: story above)

~~**Bonus (follow up ticket?):**~~
~~- [ ] Subject headings display as full strings~~

**Comment by martinlovell** [comment]
See Example Metadata: https://metadata-api.library.yale.edu/metadatacloud/api/ils/bib/5555?mediaType=json

Each Parent may have 1 or more SubjectHeading values in the json document.
The Subject Heading components are separated with " > ".

The goal is to display each Subject Heading in Blacklight so that clicking on one of the subject components will find all documents which contain a subject heading matching the subject components up to the one that is clicked.

So, in the example, one of the subject headings is Music > England > London > History and criticism.
This would display in Blacklight with each of the components being a link.

If a user clicks on London, the results will be all documents that have a subject heading starting with Music > England > London. (The subject headings may be longer, but must start with that value.)

Eric had implemented a similar functionality in QS by indexing each prefix in Solr in a multi value string field used for the facet.  (_ssim in DCS).
So for the example of Music > England > London > History and criticism, the Solr document should contain the following values for this **subject_heading_facet_ssim** field:
```
Music > England > London > History and criticism
Music > England > London
Music > England
Music
```
Do this for each subject heading in the JSON document.

The full subject headings should go into a separate field (**subject_heading_ssim**), containing just the complete subject heading:
`Music > England > London > History and criticism`

In blacklight, the display helper for the **subject_heading_ssim** show field, should split by > and make each component a link to the **subject_heading_facet_ssim** with the full heading up to that point.

**Bonus Acceptance:**
Stakeholders would like it if there was a visual clue that the links will consider the entire subject heading up to that point.  With the links, if a user hovers over London in the example, it would be idea if there the entire subject heading up to that point got the hover state.  So, when the user hovers over London, the string "Music > England > London " would be highlighted so the user understands that they aren't just searching for London as a separate item, and that they will only get results with the entire subject heading up to that point.

**Comment by martinlovell** [comment]
Needs ticket for Blacklight side.
PR for Management side: https://github.com/yalelibrary/yul-dc-management/pull/862

**Comment by FCRodriguez7** [comment]
Released management v2.55.6
